dockerfile/activemq/Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM openjdk:8u111-jdk-alpine
+MAINTAINER dev@wso2.org
 
-WORKDIR /opt
+WORKDIR /mnt
```

Comment:
I understand that for a while we have been using /mnt for installing WSO2 software on Linux environments. Nevertheless, wouldn't it be better to use /opt according to filesystem hierarchy standard?

https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard

Reply:
Yes, +1 for /opt . I started this in v3. We should keep it.

Reply:
+1 Will keep `/opt` as the standard from now on.